webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/style.scss

```diff
@@ -6,4 +6,16 @@ ul#dropdown_send_post_options {
             font-weight: bold;
         }
     }
+
+    li.core-menu-options{
+        display: flex;
+        flex-direction: column;
+        align-items: baseline;
+
+        .trailing-elements {
```

Comment:
I'm not sure how I feel about changing the style of this one menu to make the trailing elements be on a new line, but I don't have any better suggestions for how to do this at the moment 😅

Reply:
yeah, this is something that should be defined at a component level, maybe a new param to determine where to place it. But I can tell you that for a medium to long text, it won't fit in the menu placing it inline as it is right now by default.